entproto/cmd/protoc-gen-entgrpc/converter.go

```diff
@@ -90,29 +92,67 @@ func (g *serviceGenerator) newConverter(fld *entproto.FieldMappingDescriptor) (*
 	}
 	return out, nil
 }
+func convertPbMessageType(fqn, entFieldType string, conv *converter) error {
+	switch {
+	case fqn == "google.protobuf.Timestamp":
+		conv.toProtoConstructor = protogen.GoImportPath("google.golang.org/protobuf/types/known/timestamppb").Ident("New")
+	case isWrapperType(fqn):
+		typ := strings.Split(fqn, ".")[2]
+		constructor := strings.TrimSuffix(typ, "Value")
+		conv.toProtoConstructor = protogen.GoImportPath("google.golang.org/protobuf/types/known/wrapperspb").Ident(constructor)
+		goType := wrapperPrimitives[fqn]
+		if entFieldType != goType {
+			conv.toProtoConversion = goType
+		}
```

Comment:
```suggestion
		if goType := wrapperPrimitives[fqn]; entFieldType != goType {
			conv.toProtoConversion = goType
		}
```

Cleaner, as it's used only in the if's scope.

Reply:
<3